src/libs/antares/study/area/list.cpp

```diff
@@ -332,8 +332,7 @@ bool saveAreaAdequacyPatchIniFile(const Area& area, const Clob& buffer)
 {
     IniFile ini;
     IniFile::Section* section = ini.addSection("adequacy-patch");
-    bool bUseAdequacyPatch = false; //adq: this should be later replaced with area.bUseAdequacyPatch
-    section->add("use-adequacy-patch", static_cast<bool>(bUseAdequacyPatch));
+    section->add("use-adequacy-patch", static_cast<bool>(area.bUseAdequacyPatch));
```

Comment:
static cast not required see [here](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/532#discussion_r793292520)